resolwe/test_helpers/test_runner.py

```diff
@@ -515,17 +543,41 @@ def resolve_process_tags(self, files):
         for proc_path in processes_paths:
             process_schemas.extend(self.find_schemas(proc_path))
 
+        # Switch to source branch and get all the schemas from there as well, since
+        # some schemas might have been removed.
+        with self.git_branch(self.only_changes_to):
+            for proc_path in processes_paths:
+                process_schemas.extend(self.find_schemas(proc_path))
+
         dependencies = self.find_dependencies(process_schemas)
         processes = set()
 
-        for filename in files:
+        def load_process_slugs(filename):
+            """Add all process slugs from specified file."""
             with open(filename, 'r') as process_file:
                 data = yaml.load(process_file)
 
                 for process in data:
                     # Add all process slugs.
                     processes.add(process['slug'])
 
+        for filename in files:
+            try:
+                load_process_slugs(filename)
+            except FileNotFoundError:
+                # File was removed, so we will handle it below when we check the
```

Comment:
Same as above, can you wrap the comments at the same wrap limit as the surrounding code?